os_migrate/plugins/module_utils/server_port.py

```diff
@@ -62,7 +62,10 @@ class ServerPort(resource.Resource):
         'fixed_ips_refs',
         'network_ref',
     ]
-    sdk_params_from_params = []
+    sdk_params_from_params = [
+        'id',
+        'binding_profile'
+    ]
```

Comment:
I'd try with
```
params_from_sdk = [
        'binding_profile'
]
```

We don't need `id` (port id) here. We only need `network_id` and that is already being fed in because it's listed in `sdk_params_from_refs`.